core/src/main/java/org/fao/geonet/kernel/search/LuceneSearcher.java

```diff
@@ -329,48 +322,35 @@ public static LanguageSelection determineLanguage(
                 try {
                     if (test.length() > 0) {
                         String detectedLanguage = LanguageDetector.getInstance().detect(srvContext, test.toString());
-                        if (Log.isDebugEnabled(Geonet.LUCENE)) {
-                            Log.debug(Geonet.LUCENE, "automatic language detection: '" + test + "' is in language " + detectedLanguage);
-                        }
+                        LOGGER.debug( "automatic language detection: '{}' is in language {}", test, detectedLanguage);
                         finalDetectedLanguage = detectedLanguage;
                         detected = true;
                     }
                 } catch (Exception x) {
-                    Log.error(Geonet.LUCENE, "Error auto-detecting language: " + x.getMessage());
+                    LOGGER.error("Error auto-detecting language: {}", x.getMessage());
                     x.printStackTrace();
```

Comment:
Not related to this PR particularly, but printStackTrace, as well as System.[err,out].println should be avoided

Reply:
Completely agree... Not sure if I should add it here, as in theory I am not changing any code.